src/services/fileStorage.tsx

```diff
@@ -0,0 +1,45 @@
+/**********************************
+  
+ TODO: Will be moved to backend
+
+**********************************/
+
+import AWS from 'aws-sdk'
+
+//check clickup for the credentials
+const bucketName = '';
```

Comment:
Thanks for keeping these values private! 🙏 

As having env variables might not be as straightforward as in React, and we don't have to go into that.
We could have a single file to hold all the variables that we will need for the app.

For this, I suggest creating a file named config in the root of the src folder.

```
const config = {
S3:{
 bucketName = '',
 accessKeyId =  '',
 secretAccessKey = '',
 region = '',
},
//.... Other variables we might need

}

export default config

```

and as you did, we can be careful not to add the values but this will help us have all the keys in one file.

Then in this file, you can import them

```
import config from 'src/config'

const bucketName = config.S3.bucketName;
const accessKeyId =  config.S3.accessKeyId;
const secretAccessKey = config.S3.secretAccessKey;
const region = config.S3.region;
```

Let me know what you think :pray:

Reply:
Thank you for the feedback @whamester. I added config.ts to gitignore. It contains the configuration on frontend, like what you suggested.